src/worker/views/news.ts

```diff
@@ -127,10 +127,28 @@ export const processEvents = async (
 				{ pid: event.pids[0] },
 				"noCopyCache",
 			);
+			let team;
+			if (event.tid) {
+				team = await idb.getCopy.teamsPlus(
+					{
+						attrs: ["colors"],
+						tid: event.tid,
+						// season: event.season,
+					},
+					"noCopyCache",
+				);
+			}
```

Comment:
For this change...

- Would be better to use the `getTeamInfoBySeason` function, because that can get the jersey colors for a specific season, in the case that they changed over time. Since past seasons can be viewed on this page.
- Put team colors on the `event` object, rather than `p.face`, I think that is clearer. Which could be done in `processEvents`.
- In `processEvents`, it should find all the tids needed to display this page (either from tid parameter, or by scanning all events if tid is undefined), and then get the colors for all of them up front, rather than doing it separately for each event. Since `getTeamInfoBySeason` has to read from the hard drive for past seasons.
- Also make a similar change in Headlines.tsx for the dashboard. We should also make sure there's no performance issue from using `getTeamInfoBySeason` from the dashboard - maybe add a check up front for if it's the current season, and then do `await idb.cache.teamSeasons.indexGet("teamSeasonsByTidSeason", [tid, g.get("season")]);` if it is, which bypasses hitting the hard drive.

If that sounds like too much, then maybe take the news feed changes out of this PR, and copy the above text to another issue for later :) honestly that might be better to have a separate PR, since it's a little complicated.